src/components/shared/painting-info-template/painting-info-template.jsx

```diff
@@ -0,0 +1,115 @@
+import Image from 'next/image';
+import styles from './painting-info-template.module.scss';
+import PrimaryButton from '@/components/shared/primary-button';
+import RichText from '@/components/shared/rich-text';
+import { formatPrice } from '@/utils/string-utils';
+import { BLOCKS, MARKS } from '@contentful/rich-text-types';
+import { useRef, useState } from 'react';
+
+export default function PaintingInfoTemplate({
+  imageCollection,
+  collection,
+  name,
+  description,
+  additionalDescription,
+  price,
+  details = [],
+}) {
+  const [showAdditionalDescription, setShowAdditionalDescription] =
+    useState(false);
+  const additionalDescriptionRef = useRef();
+  const { url, width, height, description: alt } = imageCollection.items[0];
+
+  const richTextConfig = {
+    renderMark: {
+      [MARKS.BOLD]: text => <span className={styles.richTextBold}>{text}</span>,
+      [MARKS.ITALIC]: text => (
+        <span className={styles.richTextItalic}>{text}</span>
+      ),
+    },
+    renderNode: {
+      [BLOCKS.PARAGRAPH]: (node, children) => {
+        console.log('children', children);
+        return <p className={styles.richText}>{children}</p>;
+      },
+    },
+  };
+
+  function handleClickReadMore() {
+    setShowAdditionalDescription(!showAdditionalDescription);
+    if (!showAdditionalDescription) {
```

Comment:
I'm not sure if it would be wise to have "Show less" rendered if the showAdditionalDescription is set to true. That way we can collapse the description down to its original form.